js/generate.js

```diff
@@ -0,0 +1,31 @@
+var entries = [];
+
+function Entry (opts) {
+  this.sitename = opts.sitename;
+  this.siteUrl = opts.siteUrl;
+  this.description = opts.description;
+}
+
+Entry.prototype.toHtml = function() {
+  var $newEntry = $('article.template').clone();
+  // var entryTitle = $newEntry.find('#site');
+  // entryTitle.text(this.sitename);
+  var newSiteLink = '<a href="' + this.siteUrl + '">' + this.sitename + '</a>';
+  console.log(newSiteLink);
+  $newEntry.find('h1').html(newSiteLink);
+  // $newEntry.find('#site').text(this.sitename);
+  // $newEntry.find('#site > a').attr('href, this.siteUrl');
```

Comment:
Try and avoid leaving commented out code when pulling into the master branch. It makes it harder for others to read and understand what your intentions are. If you have a good reason to leave it here, try and leave a comment as to why it is commented out.